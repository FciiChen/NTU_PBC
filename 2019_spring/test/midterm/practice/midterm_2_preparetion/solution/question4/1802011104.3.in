給定一個句子，所有的跳連詞為中間間隔任意非標點符號的兩個字元。，如 “誡以愛民、納諫二事”中的跳連詞為 、誡_愛、以_民、納_二、諫_事。寫一個程式，由使用者讀入一段文章，然後計算所有的跳連詞出現的次數，依照次數由大到小排列，並輸出最常見的十個詞與其頻率，中間以一個空白分隔，每行一筆資料。如果頻率一樣，則依照雙連詞的內部編碼由小到大排序。
讀取文章時，輸入可能會超過一行，你的程式應持續讀入，直到使用者輸入 “BREAK”為止。在計算跳連詞時，應跳過所有標點符號，如 “（）：「，。？」…”(包含全形引號，前列標點符號並非完整列示) 都不應該出現在跳連詞中(包含中間跳掉的那個字)。依照中文的特性，空白(全形或半形)與換行亦當作是標點符號。
BREAK