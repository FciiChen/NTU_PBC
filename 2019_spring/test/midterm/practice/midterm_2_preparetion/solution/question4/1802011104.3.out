跳_詞 5
標_符 4
點_號 4
的_個 3
的_連 3
中_跳 2
使_者 2
到_排 2
所_的 2
有_跳 2
